devel/unicorn/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        unicorn-engine unicorn 2.0.0-rc7
+github.tarball_from archive
+
+name                unicorn
+version             2.0.0-rc7
+license             GPL-2+
+categories          devel
+platforms           darwin
+maintainers         {@crowell crowell.biz:jeff} openmaintainer
+description         CPU emulator framework
+long_description    Unicorn is a lightweight multi-platform, \
+    multi-architecture CPU emulator framework.
+homepage            https://www.unicorn-engine.org/
+checksums           rmd160  30c6ee76d26ea9cc97c7d4800d84de9f2d082084 \
+                    sha256  f92f267e305b562bd12cfe9df56f4ec37771624fbf62c7cc265b697f53b62c45 \
+                    size    3984228
+
+depends_build       port:cmake \
```

Comment:
since you're including the `cmake` PortGroup you do not need to add a dependency on `cmake` anymore, that is done automactically